lms/djangoapps/student_account/views.py

```diff
@@ -3,28 +3,35 @@
 import logging
 import json
 import urlparse
+from datetime import datetime
 
 from django.conf import settings
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required
+from django.core.urlresolvers import reverse, resolve
 from django.http import (
-    HttpResponse, HttpResponseBadRequest, HttpResponseForbidden
+    HttpResponse, HttpResponseBadRequest, HttpResponseForbidden, HttpRequest
 )
 from django.shortcuts import redirect
-from django.http import HttpRequest
-from django_countries import countries
-from django.core.urlresolvers import reverse, resolve
 from django.utils.translation import ugettext as _
 from django.views.decorators.csrf import ensure_csrf_cookie
 from django.views.decorators.http import require_http_methods
-
-from lang_pref.api import released_languages, all_languages
+from django_countries import countries
 from edxmako.shortcuts import render_to_response
+import pytz
 
+from commerce.models import CommerceConfiguration
 from external_auth.login_and_register import (
     login as external_auth_login,
     register as external_auth_register
 )
+from lang_pref.api import released_languages, all_languages
+from openedx.core.djangoapps.commerce.utils import ecommerce_api_client
+from openedx.core.djangoapps.programs.models import ProgramsApiConfig
```

Comment:
This is imported but no longer used. @ahsan-ul-haq what was the reason for importing this in the first place? Is it necessary to keep the programs tab in the header?